### PR TITLE
ROX-20816: Release notes for RHACS 4.3

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):
- merge to `rhacs-docs`
- cherry pick to `rhacs-docs-4.3`

[Issue](https://issues.redhat.com/browse/ROX-20816)

[Link to docs preview
](https://67801--docspreview.netlify.app/openshift-acs/latest/release_notes/43-release-notes)

QE review:
- [x] QE has approved this change. (ACS has no QE - see comments for SME approvals.)

Additional information:

### CODE REVIEW

File: release_notes/43-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+[id="hardware-software-support_{context}"]
+=== Increased support for hardware and software
+
+{product-title-short} support has been increased for the following hardware and software:
+
+* {rh-ibm-cloud}: You can now protect {rh-ibm-cloud} by using {product-title-short}. With this release, you can run secured clusters on {rh-ibm-cloud}.
+* {ibm-power} and {ibm-z}: {product-title-short} Central Services are now supported on {ibm-power} and {ibm-z}.
+* {product-rosa}: {product-title-short} is supported on {product-rosa-short} hosted control plane (HCP) enabled clusters.
+//We have questions about whether 4.14 is supported and https://issues.redhat.com/browse/ROX-17348 seems to indicate that it is supported - should I leave this in?
```
Comment (kcarmichael08) [inline release_notes/43-release-notes.adoc:40]: Reviewers: please note outstanding question here about OCP 4.14 support.

**Comment by kcarmichael08** [comment]
PM approvals: Shubha approved Vuln Management info via Slack. Doron approved Cloud Service info in meeting.

**Review by skrthomas — COMMENTED** [review]
Sorry!! I got side-tracked with a re-review of something!! I left a few comments for you. Looks great. Lots of good work here!!!!!

File: release_notes/43-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,276 @@
+:_content-type: ASSEMBLY
+[id="release-notes-43"]
+= {product-title} 4.3
+include::modules/common-attributes.adoc[]
+:context: release-notes-43
+
+toc::[]
+
+{rh-rhacs-first} is an enterprise-ready, Kubernetes-native container security solution that protects your vital applications across the build, deploy, and runtime stages of the application lifecycle.
+{product-title} deploys into your infrastructure and integrates with your DevOps tools and workflows. This integration provides better security and compliance, enabling DevOps and InfoSec teams to operationalize security.
```
[the remark this answers is not in the document]
Comment (kcarmichael08) [reply release_notes/43-release-notes.adoc:10]: I'll talk to marketing and see if we can come up with a better term. (It's marketing's boilerplate, so I'm hesitant to change without discussing with them first.)

File: release_notes/43-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+| DEP
+| DEP
+| REM
+
+| Examining images for Application-level dependencies for vulnerability reporting: `dotnet/shared/Microsoft.AspNetCore.App/` and `dotnet/shared/Microsoft.NETCore.App/`
+| DEP
+| DEP
+| REM
+
+| `Expiration` field in `Exclusion` proto
```
[the remark this answers is not in the document]
Comment (kcarmichael08) [reply release_notes/43-release-notes.adoc:137]: I've seen it in Kubernetes docs, but I don't know. It has been in the RNs for the last couple of releases like this, so I'd prefer to leave it to be consistent. (And maybe ask for future docs to make sure that I shouldn't be using a different word.)

File: release_notes/43-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+| Stores image scan results and vulnerability definitions.
+a|`registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8:4.3.0`
+
+| Collector
+| Collects runtime activity in Kubernetes or {ocp} clusters.
+a| * `registry.redhat.io/advanced-cluster-security/rhacs-collector-rhel8:4.3.0`
+* `registry.redhat.io/advanced-cluster-security/rhacs-collector-slim-rhel8:4.3.0`
+
+| Central DB
+| Postgres instance that provides the database storage for Central.
```
[the remark this answers is not in the document]
Comment (kcarmichael08) [reply release_notes/43-release-notes.adoc:274]: Good catch!

**Pull request merged**